tsconfig-scryptTS.json

```diff
@@ -5,6 +5,13 @@
     "noEmit": true,
     "experimentalDecorators": true,
     "target": "ESNext",
-    "esModuleInterop": true
+    "esModuleInterop": true,
+    "plugins": [
```

Comment:
Is this intended? Looks like a local config.

Reply:
Hmmm. No, and I didn't see that in the changelog when I submitted that.

Reply:
Will merge once this part is removed.